packages/df-blogs/src/ViewComment.tsx

```diff
@@ -0,0 +1,93 @@
+import React from 'react'
+import { Comment as SuiComment } from 'semantic-ui-react'
+
+import { withCalls, withMulti } from '@polkadot/ui-api/with';
+import { Option } from '@polkadot/types';
+
+import { PostId, CommentId, Comment } from './types';
+import { queryBlogsToProp } from './utils';
+import { MyAccountProps, withMyAccount } from '@polkadot/joy-utils/MyAccount';
+import Section from '@polkadot/joy-utils/Section';
+
+import { NewComment } from './EditComment';
+import { AddressMini } from '@polkadot/ui-app';
+
+
+type Props = MyAccountProps & {
+  postId: PostId,
+  commentIds?: CommentId[]
+};
+
+function InnerCommentsByPost(props: Props) {
+
+  const {
+    myAddress,
+    postId,
+    commentIds = []
+  } = props;
+
+  const commentsCount = commentIds ? commentIds.length : 0;
+
+  if (!commentIds || commentIds.length === 0) {
```

Comment:
Delete this if-block. You can show that there are no comments in the next return block with `a ? b : c` operator.